web/src/main/java/ua/com/vertex/context/SessionListener.java

```diff
@@ -0,0 +1,21 @@
+package ua.com.vertex.context;
+
+import org.springframework.stereotype.Component;
+
+import javax.servlet.http.HttpSessionEvent;
+import javax.servlet.http.HttpSessionListener;
+
+@Component
+public class SessionListener implements HttpSessionListener {
+
+    @Override
+    public void sessionCreated(HttpSessionEvent event) {
+        System.out.println("sessionCreated");
+        event.getSession().setMaxInactiveInterval(30 * 60);
```

Comment:
Ok. Changed. Seems to be working. In fact, I do not like it. And actually, this is the only project where I have a web.xml at all. That's why I prefer creation of a listener or, in case of Boot, application.properties